aten/src/ATen/native/TensorAdvancedIndexing.h

```diff
@@ -6,7 +6,9 @@
 #include <ATen/core/Tensor.h>
 #include <ATen/native/DispatchStub.h>
 #include <ATen/native/ReductionType.h>
-#include <ATen/native/cpu/radix_sort.h>
+#ifdef USE_FBGEMM
```

Comment:
seems like this include is causing a build failure (see https://hud.pytorch.org/pr/103074)

Reply:
Fixed include error, though I see now some errors in `test_torchinductor_dynamic_shapes` - probably not related to my change, could you please confirm?